webpack.config.js

```diff
@@ -1,164 +1,164 @@
+          if ( chunk.name.match( /\/modules\// ) ) {
+            return `${ chunk.name.replace( '/js/', '/css/' ) }.css`;
+          }
+
+          return '../css/[name].css';
         },
-        modules: [
-            path.resolve('./node_modules'),
-            path.resolve(path.join(__dirname, 'src/')),
-        ]
-    },
-
-    externals: {
-        jquery: 'jQuery',
-        'chart.js': 'Chart',
-        moment: 'moment'
-    },
-
-    plugins,
-
-    module: {
-        rules: [
-            {
-                test: /\.js$/,
-                exclude: /node_modules/,
-                use: [
-                    {
-                        loader: require.resolve( 'babel-loader' ),
-                        options: {
-                            // Babel uses a directory within local node_modules
-                            // by default. Use the environment variable option
-                            // to enable more persistent caching.
-                            cacheDirectory: process.env.BABEL_CACHE_DIRECTORY || true,
-                        },
-                    },
-                ],
-            },
-            {
-                test: /\.vue$/,
-                loader: 'vue-loader',
-                options: {
-                    extractCSS: true
-                }
+      }
+    ),
+
+    new VueLoaderPlugin(),
+  ],
+
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        loader: 'babel-loader',
+      },
+      {
+        test: /\.vue$/,
+        loader: 'vue-loader',
+      },
+      {
+        test: /\.(less|css)$/i,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: "css-loader",
+            options: {
+              sourceMap: ! isProduction,
             },
-            {
-                test: /\.less$/,
-                use: extractCss.extract({
-                    use: [{
-                        loader: "css-loader"
-                    }, {
-                        loader: "less-loader"
-                    }]
-                })
+          },
+          {
+            loader: "less-loader",
+            options: {
+              sourceMap: ! isProduction,
             },
-            {
-                test: /\.css$/,
-                use: [ 'style-loader', 'css-loader' ]
-            }
-        ]
+          },
+        ],
+      },
+      {
+        test: /\.(bmp|png|jpe?g|gif|webp)$/i,
+        type: 'asset/resource',
+        generator: {
+          filename: '../images/[name][ext][query]'
+        },
+      },
+      {
+        test: /\.svg/,
+        type: 'asset/inline'
+      },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/i,
+        type: 'asset/resource',
+        generator: {
+          filename: '../font/[name].[ext]',
+        },
+      },
+    ]
+  },
+}
+
+if (!isProduction) {
+  updatedConfig.devServer = {
+    devMiddleware: {
+      writeToDisk: true,
     },
+    allowedHosts: 'all',
+    host: 'localhost',
```

Comment:
kindly get this from the environment variable: https://github.com/weDevsOfficial/webot/blob/master/webpack.config.js